climada/test/test_plot.py

```diff
@@ -25,16 +25,26 @@
 import contextily as ctx
 import urllib
 
+from rasterio.windows import Window
 from climada.entity.entity_def import Entity
 from climada.hazard.base import Hazard
 from climada.entity.exposures.base import Exposures
 from climada.entity.impact_funcs.impact_func_set import ImpactFuncSet
 from climada.engine import ImpactCalc, ImpactFreqCurve
-from climada.util.constants import HAZ_DEMO_MAT, ENT_DEMO_TODAY
+from climada.util.constants import HAZ_DEMO_FL, HAZ_DEMO_MAT, ENT_DEMO_TODAY
+
+# Define an exposure to use for plotting
+exp = Exposures.from_raster(HAZ_DEMO_FL, window= Window(10, 20, 50, 60))
```

Comment:
This creates a single exposure object for the entire script. Please move this code into a test function or remove it altogether.